engine/access/rpc/backend/backend_test.go

```diff
@@ -210,9 +210,9 @@ func (suite *Suite) TestGetCollection() {
 	suite.assertAllExpectations()
 }
 
-// TestTransactionStatusTransition tests that the status of transaction changes from Finalized to Sealed
+// TestTransactionFinalizedToSealedStatusTransition tests that the status of transaction changes from Finalized to Sealed
```

Comment:
`TestTransactionStatusTransition` renamed to `TestTransactionFinalizedToSealedStatusTransition`
`TestTransactionExpiredStatusTransition` renamed to `TestTransactionPendingToExpiredStatusTransition `
Added new test `TestTransactionPendingToFinalizedStatusTransition`